lessons/codap-bar-charts/langs/en-us/pages/plot-practice-2.adoc

```diff
@@ -8,29 +8,29 @@ p { margin: 0 !important; }
 </style>
 ++++
 
-Fill in the tables, predict what the graph will look like (a rough sketch is fine!), and then use CODAP to make the following displays.
+Use the starter-file{animals} to create the following displays in CODAP. Then, sketch the displays and answer the questions.
 
 @vspace{1ex}
 
 @n A histogram of the number of `pounds` that animals weigh.
-[cols="^1a,^1a,^1a", options="header"]
+[cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Configuration*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Configuration*}
```

Comment:
NO. MORE. 

@center{...} please! We made this decision almost a year ago, darnit!!!

Reply:
Sorry! I have mostly used @center for images. I think that's why it didn't occur to me. (I copied the @span thing from another place on the same page.)